README.md

```diff
@@ -187,6 +187,11 @@ more about it.
 
 **.regal/config.yaml**
 ```yaml
+# Files can be excluded from all lint 
+# rules according to glob-patterns
+ignore:
```

Comment:
I wonder if we should do:
```yaml
ignore:
  files:
    - file1.rego
    - "*_tmp.rego"
```
I.e. add a `files` attribute under `ignore`, in case we want to add something similar for packages (or whatnot) later?

Reply:
Yes, I like that idea much better! Will implement :)

Reply:
Awesome!